src/dotnet/APIView/APIViewWeb/Managers/PullRequestManager.cs

```diff
@@ -49,14 +55,14 @@ public PullRequestManager(
             _telemetryClient = telemetryClient;
             _languageServices = languageServices;
             _logger = logger;
+            _gitHubClientFactory = gitHubClientFactory;
 
-            var ghToken = _configuration["github-access-token"];
-            if (!string.IsNullOrEmpty(ghToken))
-            {
-                _githubClient.Credentials = new Credentials(ghToken);
-                _isGitClientAvailable = true;
-            }
-            var pullRequestReviewCloseAfter = _configuration["pull-request-review-close-after-days"] ?? "30";
+            string appId = _configuration["GitHubApp:Id"];
```

Comment:
Is it interesting to read these variables here any longer?

Reply:
I don't think so! removing them